ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1.admin;
+
+option java_outer_classname = "UserManagementServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1.admin";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
+
+
+
+// --> See USER_MANAGEMENT_DESIGN.md for status and notes <-- //
+
+
+
+// Manage users and their rights for interacting with the Ledger API served by participant node.
+//
+// The API roughly follows the Google API style guidelines for resource-oriented design
+// (https://cloud.google.com/apis/design/resources) with the following simplifications to 
+// ease its implementation:
+// 
+// 1. List methods do not (yet) support pagination, 
+//    as we expect to have fewer than 10k users and 1k rights per user.
+// 
+// 2. Resources are not named as per the Google API style guide, 
+//    as this would be in contrast with the other Ledger API services.
+//
+service UserManagementService {
```

Comment:
I think adding the required auth claims here would be quite helpful.

Reply:
Would this API be needed at all if we did the user management on-ledger?

Reply:
added the auth claims in https://github.com/digital-asset/daml/pull/11818/commits/b9cb2c6914380cabd286320a4956bd34c5ee8062; and answered to @bame-da 's comment in https://github.com/digital-asset/daml/pull/11818#issuecomment-976702106